Matplotlib created a temporary config/cache directory at /tmp/matplotlib-4_w7om4o because the default path (/home/mriqc/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-06pb84b9 because the default path (/home/mriqc/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-mz0bhnij because the default path (/home/mriqc/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Traceback (most recent call last):
  File "/opt/conda/bin/mriqc", line 8, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.9/site-packages/mriqc/cli/run.py", line 167, in main
    mriqc_wf.run(**_plugin)
  File "/opt/conda/lib/python3.9/site-packages/nipype/pipeline/engine/workflows.py", line 638, in run
    runner.run(execgraph, updatehash=updatehash, config=self.config)
  File "/opt/conda/lib/python3.9/site-packages/mriqc/engine/plugin.py", line 184, in run
    self._clean_queue(jobid, graph, result=result)
  File "/opt/conda/lib/python3.9/site-packages/mriqc/engine/plugin.py", line 256, in _clean_queue
    raise RuntimeError("".join(result["traceback"]))
RuntimeError: Traceback (most recent call last):
  File "/opt/conda/lib/python3.9/site-packages/mriqc/engine/plugin.py", line 60, in run_node
    result["result"] = node.run(updatehash=updatehash)
  File "/opt/conda/lib/python3.9/site-packages/nipype/pipeline/engine/nodes.py", line 527, in run
    result = self._run_interface(execute=True)
  File "/opt/conda/lib/python3.9/site-packages/nipype/pipeline/engine/nodes.py", line 645, in _run_interface
    return self._run_command(execute)
  File "/opt/conda/lib/python3.9/site-packages/nipype/pipeline/engine/nodes.py", line 771, in _run_command
    raise NodeExecutionError(msg)
nipype.pipeline.engine.nodes.NodeExecutionError: Exception raised while executing Node synthstrip.

Cmdline:
	synthstrip -b 1 -i /scratch1/03201/jbwexler/work_dir/mriqc/ds004056_sub-29/mriqc_wf/anatMRIQC/synthstrip_wf/_in_file_..scratch1..03201..jbwexler..openneuro_derivatives..derivatives..mriqc..ds004056-mriqc..sourcedata..raw..sub-29..ses-02..anat..sub-29_ses-02_acq-02_T1w.nii.gz/pre_n4/clipped_corrected.nii.gz --model /opt/freesurfer/models/synthstrip.1.pt -o clipped_corrected_desc-brain.nii.gz -m clipped_corrected_desc-brain_mask.nii.gz
Stdout:
	Configuring model on the CPU
	Using custom model weights
	Input image read from: /scratch1/03201/jbwexler/work_dir/mriqc/ds004056_sub-29/mriqc_wf/anatMRIQC/synthstrip_wf/_in_file_..scratch1..03201..jbwexler..openneuro_derivatives..derivatives..mriqc..ds004056-mriqc..sourcedata..raw..sub-29..ses-02..anat..sub-29_ses-02_acq-02_T1w.nii.gz/pre_n4/clipped_corrected.nii.gz
Stderr:
	Traceback (most recent call last):
	  File "/opt/conda/bin/synthstrip", line 8, in <module>
	    sys.exit(main())
	  File "/opt/conda/lib/python3.9/site-packages/mriqc/synthstrip/cli.py", line 125, in main
	    conformed = conform(image)
	  File "/opt/conda/lib/python3.9/site-packages/mriqc/synthstrip/cli.py", line 224, in conform
	    nii = Affine(
	  File "/opt/conda/lib/python3.9/site-packages/nitransforms/base.py", line 282, in apply
	    _as_homogeneous(self.map(_ref.ndcoords.T), dim=_ref.ndim)
	  File "/opt/conda/lib/python3.9/site-packages/nitransforms/base.py", line 141, in ndcoords
	    np.vstack((self.ndindex, np.ones((1, self._npoints)))),
	  File "<__array_function__ internals>", line 180, in vstack
	  File "/opt/conda/lib/python3.9/site-packages/numpy/core/shape_base.py", line 282, in vstack
	    return _nx.concatenate(arrs, 0)
	  File "<__array_function__ internals>", line 180, in concatenate
	numpy.core._exceptions._ArrayMemoryError: Unable to allocate 384. MiB for an array with shape (4, 12582912) and data type float64
Traceback:
	RuntimeError: subprocess exited with code 1.

MELODIC outputs not found, assuming it didn't converge.
